guides/release/accessibility/learning-resources.md

```diff
@@ -1,31 +1,32 @@
-- [Web Content Accessibility Guidelines(WCAG) 2.1](https://www.w3.org/TR/WCAG21/)
-- [Accessible Rich Internet Applications (ARIA) 1.1](https://www.w3.org/TR/wai-aria-1.1/) 
-- [How to meet WCAG (Quick Reference)](https://www.w3.org/WAI/WCAG21/quickref/)
+*Note: Les liens ci-dessous redirigent vers des articles écrits en Anglais
 
+- [Documentation Web MDN: Accessibilité](https://developer.mozilla.org/en-US/docs/Learn/Accessibility)
+- [Utiliser ARIA:](https://www.w3.org/TR/using-aria/) un guide pratique pour développeurs.euses: Comment ajouter des informations d'accessibilité sur des éléments HTML
+- [Web Content Accessibility Guidelines(WCAG): Directives du contenu web sur l'accessibilité 2.1](https://www.w3.org/TR/WCAG21/)
+- [Accessible Rich Internet Applications (ARIA): Applications internet riches et accessibles 1.1](https://www.w3.org/TR/wai-aria-1.1/)
+- [Comment répondre aux directives WCAG (référence rapide)](https://www.w3.org/WAI/WCAG21/quickref/)
 
-## Practical Resources
+## Ressources pratiques
 
-- [Accessibility Insights](https://accessibilityinsights.io/)- Use FastPass to find common, high-impact issues 
-- [aXe extension for Chrome](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd)
-- [Accessibility Support](https://a11ysupport.io/)- Find out if your code will work with assistive tech
-- [How and where to report accessibility bugs](https://www.digitala11y.com/how-where-to-report-accessibility-bugs/)
+- [Aperçus sur l'accessibilité](https://accessibilityinsights.io/)- Utiliser FastPass pour trouver des problèmes communs à fort impacts
+- [extension aXe pour Chrome](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd)
+- [Aide sur l'accessibilité](https://a11ysupport.io/)- Comprendre si son code marche avec les technologies d'assistance
+- [Où et comment ajouter un bug d'accessibilité](https://www.digitala11y.com/how-where-to-report-accessibility-bugs/)
 
 
-### Design 
+### Design
 
-- [ColorSafe](http://colorsafe.co/)- Empowering designers with beautiful and accessible color palettes based on WCAG Guidelines of text and background contrast ratios
-- [Accessible Color Palette Builder](https://toolness.github.io/accessible-color-matrix/)
+- [ColorSafe](http://colorsafe.co/)- Aider les designers avec de magnifiques palettes de couleurs accessibles, basées sur les directives WCAG, ratio de couleurs de contraste de textes et arrières plans.
+- [Créateur de palettes de couleur accessibles](https://toolness.github.io/accessible-color-matrix/)
 
 
-### Writing
+### Ecriture
 
-- [Plain Language](https://plainlanguage.gov/) - Learn how to write in a way that makes it easier for people to read and understand. 
+- [Langage clair et simple](https://plainlanguage.gov/) - Apprendre comment écrire d'une manière accessible et inclusive pour une compréhension de lecture plus facile.
```

Comment:
👇 In that case I would replace EmberGuides invented title with the actual web site title.
```suggestion
- [plainlanguage.gov](https://plainlanguage.gov/) - Apprendre comment écrire d'une manière accessible et inclusive pour une compréhension de lecture plus facile.
```

Reply:
What about adding the french government page about accessibility to this list of links https://www.gouvernement.fr/accessibilite ?

Reply:
I tried to find official resources, but I think I don't know what term/word we use in French as the equivalent of plain language 😬 Do you know?

Reply:
I was not able to find official resources either. As always we are lacking official resources about a11y :grim

Reply:
:(